session_2/cricviz/app/models/cricketer.rb

```diff
@@ -29,12 +35,20 @@ def batting_average
   #   data is incomplete
   # - If the player has not batted yet, return nil
   def batting_strike_rate
-    raise NotImplementedError
+    if innings_batted == nil || runs_scored == nil || balls_faced == nil || balls_faced == 0
+      return nil
+    else
+      return (runs_scored * 100).to_f / balls_faced
+    end
   end
 
   # Create records for the classical batters
   def self.import_classical_batters
-    raise NotImplementedError
+    Cricketer.create(name: "Brian Lara", country: "West Indies", role: "Batter", matches: 131, innings_batted:232, not_out:6, runs_scored:11953, balls_faced: 19753, high_score:400, centuries:34, half_centuries:48).save
```

Comment:
You don't need to say `Cricketer.create(...).save`, just `Cricketer.create(...)` is enough